### PR TITLE
Split tx run + Queue txs

### PR DESCRIPTION
**Pull request by yann300**
fix https://github.com/ethereum/browser-solidity/issues/248

### CODE REVIEW

**Comment by yann300** [comment]
fix https://github.com/ethereum/remix/issues/144

File: src/universal-dapp.js [hunk 1]
```diff
@@ -34,10 +33,18 @@ function UniversalDApp (executionContext, options, txdebugger) {
   self.executionContext.event.register('contextChanged', this, function (context) {
     self.reset(self.contracts)
   })
+  self.txRunner = new TxRunner(executionContext, {
+    queueTxs: true,
+    personalMode: true
```
Comment (axic) [inline src/universal-dapp.js]: I think this will break because we've never tested personalMode.

File: src/universal-dapp.js [hunk 2]
```diff
@@ -733,7 +725,7 @@ UniversalDApp.prototype.runTx = function (args, cb) {
             return callback(err)
           }
 
-          tx.from = ret
+          tx.from = accounts[ret]
```
Comment (axic) [inline src/universal-dapp.js]: `accounts` is specific to VM mode (it contains private keys).

File: src/universal-dapp.js [hunk 3]
```diff
@@ -746,115 +738,19 @@ UniversalDApp.prototype.runTx = function (args, cb) {
             return callback('No accounts available')
           }
 
-          tx.from = ret[0]
+          tx.from = accounts[ret[0]]
```
Comment (axic) [inline src/universal-dapp.js]: Same here, this will break web3 mode.

File: src/universal-dapp.js (hunk 1 above)
Comment (yann300) [inline src/universal-dapp.js]: yep :) i have fixed that in another pr

**Pull request merged**